turbopack/crates/turbo-tasks-fs/Cargo.toml

```diff
@@ -56,6 +57,7 @@ turbo-tasks = { workspace = true }
 turbo-tasks-hash = { workspace = true }
 unicode-segmentation = { workspace = true }
 urlencoding = { workspace = true }
+lz4 = "1.28.1"
```

Comment:
We're already using the `lzzzz` crate in a few places: https://github.com/search?q=repo%3Avercel%2Fnext.js%20lzzzz&type=code

It looks like the `lz4` crate *was* unmaintained, making `lzzzz` the obvious choice, but now it looks like both are actively maintained, so it's unclear to me which one is now the better choice.

Either way, we should pick one of the two.